.github/workflows/prometheus-exporter-release.yaml

```diff
@@ -0,0 +1,94 @@
+name: Release Prometheus Prefect Exporter Helm Chart
+
+"on":
+  push:
+    tags:
+      # prefect-prometheus-exporter-2023.9.1, but not prefect-prometheus-exporter-2023.09.01
+      # prefect-prometheus-exporter-2023.11.5, but not prefect-prometheus-exporter-2023.11.05
+      # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+      - 'prefect-prometheus-exporter-2[0-9][2-9][3-9].1?[0-9].[1-3]?[0-9]'
```

Comment:
this makes sense as the release name, is it also the tag name??

Also - should we convert to doing this for the other charts. i think we used to back in the day?

Reply:
Yeah I'll create an issue to do the same w/ the other charts.  This is also the tag name, yes